.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [ 1, {
+      "js": "always", "json": "always"
+    }]
+  },
+  "ignorePatterns": [
+    "dist/",
+    "build/"
```

Comment:
- [ ] Please make sure the Nav bar matches the wireframe i.e not exactly but to be close to it 
 
| Your design | Wireframe |
| ---------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/181838995-3f0b89b7-67a2-4d7f-b9b2-ebeb99ecd52c.png) | ![image](https://user-images.githubusercontent.com/74744787/181839038-309d3e34-0bc0-47f5-99bc-3ded36965a7c.png) |

Reply:
Fixed